docs/src/pages/components/menus/menus.md

```diff
@@ -24,6 +24,18 @@ Choosing an option should immediately ideally commit the option and close the me
 
 {{"demo": "pages/components/menus/BasicMenu.js"}}
 
+## Icon menu
+
+In desktop viewport, padding is increased to give more space to the menu.
+
+{{"demo": "pages/components/menus/IconMenu.js", "bg": true}}
+
+## Dense menu
+
+For the menu that has long list and long text, use `dense` to reduce the padding (this property only affects desktop viewport).
```

Comment:
```suggestion
For the menu that has long list and long text, you can use the `dense` prop to reduce the padding (this property only affects desktop viewport).
```

Mainly about MUST vs CAN. Long lists don't need to be made dense especially since this may come at the cost of legibility which may make it even harder to comprehend long lits.